.travis.yml

```diff
@@ -11,6 +11,7 @@ env:
     global:
         # Make sure beaver is in the PATH
         - PATH="$(git config -f .gitmodules submodule.beaver.path)/bin:$PATH"
+        - UPLOAD_DESTINATION=release
```

Comment:
You could move this as a default setting to the `upload-tsunami-package` script.

Reply:
Yeah, this was done when all the logic was in the `.travis.yml` but now that this variable is only used in a separate script, I think it's more clear to move the default there too.